src/resources/resources/assets/waila/lang/ru_ru.json

```diff
@@ -1,115 +1,216 @@
+  "config.waila.plugin_minecraft.note_block.type"               : "Отображать инструмент и ноту",
+  "config.waila.plugin_minecraft.note_block.note"               : "Режим отображения нот",
+  "config.waila.plugin_minecraft.note_block.note_sharp"         : "Резкий",
+  "config.waila.plugin_minecraft.note_block.note_flat"          : "Плоский",
+  "config.waila.plugin_minecraft.note_block.int_value"          : "Число",
+  "config.waila.plugin_minecraft.pet"                           : "Питомец",
+  "config.waila.plugin_minecraft.pet.owner"                     : "Отображать хазяина",
+  "config.waila.plugin_minecraft.pet.owner_desc"                : "Только для лицензионных аккаунтов Minecraft",
+  "config.waila.plugin_minecraft.pet.hide_unknown_owner"        : "Скрыть неизвестного хозяина",
+  "config.waila.plugin_minecraft.timer"                         : "Таймеры",
+  "config.waila.plugin_minecraft.timer.grow"                    : "Отображать таймер роста мобов",
+  "config.waila.plugin_minecraft.timer.breed"                   : "Отображать таймер размножения",
+  "config.waila.plugin_minecraft.attribute"                     : "Отрибуты",
+  "config.waila.plugin_minecraft.attribute.block_position"      : "Отображать позицию блока",
+  "config.waila.plugin_minecraft.attribute.block_state"         : "Отображать состояние блока",
+  "config.waila.plugin_minecraft.attribute.entity_position"     : "Отображать сужности",
+  "config.waila.plugin_minecraft.attribute.health"              : "Отображать очки жизни",
+  "config.waila.plugin_minecraft.attribute.absorption"          : "Отображать очки поглощения",
+  "config.waila.plugin_minecraft.attribute.armor"               : "Отображать очки брони",
+  "config.waila.plugin_minecraft.attribute.compact"             : "Компактный режим",
+  "config.waila.plugin_minecraft.attribute.compact_desc"        : "Отображать очки жизней и брони цифрами а не иконками",
+  "config.waila.plugin_minecraft.attribute.icon_per_line"       : "Максимум инокон на строку",
+  "config.waila.plugin_minecraft.attribute.icon_per_line_desc"  : "Количество иконок жизни и брони в строке перед началом следующей",
+  "config.waila.plugin_minecraft.attribute.long_health_max"     : "Максимум очков жизни для отображения",
+  "config.waila.plugin_minecraft.attribute.long_health_max_desc": "Наибольшее число очков жизни для отображения в виде иконок, если у моба или игрока больше чем указано здесь, будут отображены цифры",
+  "config.waila.plugin_minecraft.attribute.long_armor_max"      : "Максимум очков брони для отображения",
+  "config.waila.plugin_minecraft.attribute.long_armor_max_desc" : "Наибольшее число очков брони для отображения в виде иконок, если у моба или игрока больше чем указано здесь, будут отображены цифры",
+
+  "config.waila.plugin_waila"                                   : "Ядро Waila",
+  "config.waila.plugin_waila.show_mod_name"                     : "Показывать названия модов",
```

Comment:
Missing translation:
```
  "config.waila.plugin_waila.show_item_mod_name"                : "Show Item Mod Names",
```

Reply:
Already added and translated these lines.